package.json

```diff
@@ -28,11 +28,17 @@
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "devDependencies": {
```

Comment:
i think this is what peer dependencies are supposed to be used for: https://nodejs.org/en/blog/npm/peer-dependencies/